### PR TITLE
JP-3020 Pin PRD versions for tests that are not testing changes in PRD

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves [JP-3020](https://jira.stsci.edu/browse/JP-3020)

This PR addresses an issue with SIAF-related tests and the PRD version changes. The PRD version is fixed for those tests that are not testing for changes.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] ~updated relevant documentation~
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by stscieisenhamer** [comment]
[regression test](https://plwishmaster.stsci.edu:8081/job/RT/job/eisenhamer-dev/295/)

File: jwst/regtest/test_stp_jw00697013.py [hunk 1]
```diff
@@ -121,7 +121,9 @@ def databases(rtdata_module):
     siaf_db, metas : `set_telescope_pointing.SiafDb`, dict
         Returns the tuple of the siaf database and all exposures meta information.
     """
-    siaf_db = siafdb.SiafDb()
+
+    # Fix the PRD. Not testing changes in PRD
```
Comment (hbushouse) [inline jwst/regtest/test_stp_jw00697013.py:125]: same comment here

File: README.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ https://jwst-pipeline.readthedocs.io/en/latest/
 
 To build the docs yourself, clone this repository and build the documentation with:
 
-    pip install -e ".[docs]"
+    pip install -e ".[docs]**
```
Comment (hbushouse) [inline README.md:185]: what's the purpose of the "**" here? doesn't it need a closing quote?

**Review by kmacdonald-stsci — APPROVED** [review]
Besides Howards comment it looks good to me.

File: README.md (hunk 2 above)
Comment (hbushouse) [inline README.md:185]: Just need to fix this one little item and it should be ready to merge.

File: README.md [hunk 3]
```diff
@@ -203,9 +203,9 @@ contact the [JWST Help Desk](https://jwsthelp.stsci.edu).
 
 ## Software vs DMS build version map
 
-| jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT<br>(JWST OPS) | CRDS_CONTEXT<br>(JWST PUB) | Date       | Notes                                         |
+| jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT<br>(JWST OPS) | CRDS_CONTEXT<br>(JWST PUB** | Date       | Notes                                         |
```
Comment (hbushouse) [inline README.md:206]: It looks like you're now missing the closing paren after "(JWST PUB"

File: README.md (hunk 2 above)
Comment (stscieisenhamer) [reply README.md:185]: That would be stray fingers wondering about the keyboard. Will fix.

**Review by hbushouse — APPROVED** [review]

**Pull request merged**